spec/rubocop/cli/cli_options_spec.rb

```diff
@@ -730,7 +730,7 @@ def full_description_of_cop(cop)
                     ' ^',
                     'example2.rb:3:1: C: Inconsistent indentation ' \
                     'detected.',
-                    'def a',
+                    "def a \e[33m...\e[0m",
```

Comment:
The reason why coloring is enabled in this case can be found in `spec_helper.rb`. There we have the setting `Rainbow.enabled = false`, but it comes after `require 'rubocop'`, which means that `ELLIPSES = Rainbow('...').yellow.freeze` has already been executed when we disable coloring.